db/schema.rb

```diff
@@ -87,6 +87,14 @@
     t.index ["user_id"], name: "index_project_members_on_user_id"
   end
 
+  create_table "project_metadata", force: :cascade do |t|
+    t.json "data", null: false
```

Comment:
What is the reason for not making a json data field on the project table itself?

Reply:
No need to make the default Project select large if it has a lot of metadata